webapp/src/Controller/Team/MiscController.php

```diff
@@ -129,6 +130,12 @@ public function homeAction(Request $request): Response
         return $this->render('team/index.html.twig', $data);
     }
 
+    #[Route(path: '/updates', methods: ['GET'], name: 'team_ajax_updates')]
```

Comment:
Given that the function contains 'unread notifications' in the name, maybe the route should be `/updates-unread-notifications`? and also not `team_ajax_updates`. Or we call the method `updates()` and we change the body to
```php
 return ['unread_clarifications' => $this->json($this->dj->getUnreadClarifications())];
```

Reply:
Sorry for the naming issue, I'll change it following your advice.